README.md

```diff
@@ -1,56 +1,111 @@
 # TUEnroll
-TUEnroll is a CLI tool designed to automate exam registrations at TU Delft.
+
+**TUEnroll** is a CLI tool designed to automate the exam registration process at **TU Delft**.
 
 It periodically checks for open exam registrations and automatically registers you when new exams are available.
 
-![TUEnroll Logo](logo.png)
+<p align="center">
+  <img src="logo.png" alt="TUEnroll Logo" />
+</p>
+
+<p align="center">
+  <a href="https://github.com/dhruvan2006/tuenroll/actions">
+    <img src="https://img.shields.io/github/actions/workflow/status/dhruvan2006/tuenroll/rust.yml?branch=main&label=Build&logo=github&logoColor=white&color=blue" alt="Build Status" />
+  </a>
+  <a href="https://opensource.org/licenses/MIT">
+    <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License" />
+  </a>
+  <a href="https://github.com/dhruvan2006/tuenroll/releases">
+    <img src="https://img.shields.io/github/v/release/dhruvan2006/tuenroll?logo=github&color=green" alt="Version" />
+  </a>
+  <a href="https://crates.io/crates/tuenroll">
+    <img src="https://img.shields.io/crates/d/tuenroll?color=orange&logo=cargo&logoColor=white" alt="Downloads" />
+  </a>
+  <a href="https://crates.io/crates/tuenroll">
+    <img src="https://img.shields.io/crates/v/tuenroll.svg?logo=cargo&logoColor=white" alt="Version" />
+  </a>
+</p>
 
 ## Installation
+
+### With Cargo
+
 Use the package manager `cargo` to install tuenroll.
 
 ```bash
 cargo install tuenroll
 ```
 
-Ubuntu:
+### With binaries
+
+#### Windows
+
+```bash
+powershell -c "irm https://raw.githubusercontent.com/dhruvan2006/tuenroll/main/install.ps1 | iex"
```

Comment:
The link should be changed to: https://raw.githubusercontent.com/dhruvan2006/tuenroll/install/install.ps1

Reply:
No, after this is merged its gonna be in `main`. Maybe there's some other better location to keep this in though, idk.